Sources/SwiftSyntax/Documentation.docc/Tutorials/SwiftSyntax By Example.tutorial

```diff
@@ -171,7 +171,7 @@
       }
       
       @Step {
-        Then use the `withStatements` method to create a source file
+        Then use the `with*` method to create a source file
         that contains the extracted array of statements.
         
         The syntax tree is immutable. The result of calling `with*` methods is
```

Comment:
Since we no longer have the family of `with` methods, we should probably remove the `*` here.

```suggestion
        The syntax tree is immutable. The result of calling `with` is
```

Reply:
I didn't know the meaning of `*`. Thank you for your suggestion!